examples/confirmation-mechanisms.js

```diff
@@ -0,0 +1,43 @@
+const { Client, CryptoTransferTransaction } = require("@hashgraph/sdk");
+
+async function main() {
+  const operatorPrivateKey = process.env.OPERATOR_KEY;
+  const operatorAccount = process.env.OPERATOR_ID;
+
+  if (operatorPrivateKey == null || operatorAccount == null) {
+    throw new Error(
+      "environment variables OPERATOR_KEY and OPERATOR_ID must be present"
+    );
+  }
+
+  const client = new Client({
```

Comment:
@Cooper-Kunz 

How do you feel about the "new style" of making Clients?

```js
const client = Client.forTestnet()
    .setOperator(operatorAccount, operatorPrivateKey);
```

I'd like to discourage people from using a single node for their network configuration.

Reply:
I'm a fan. It feels more consistent with the rest of the SDK's usage.

Is this available in recent releases?

Reply:
Yes.

Reply:
What network information does this default to for testnet & mainnet respectively? All of the address book at random, via the myhbarwallet proxy?

Reply:
@Cooper-Kunz Depends. 

On Node.js, `Client.forTestnet()` is the same as:

```js
new Client({ network: {
  "0.testnet.hedera.com:50211": "0.0.3",
  "1.testnet.hedera.com:50211": "0.0.4",
  "2.testnet.hedera.com:50211": "0.0.5",
  "3.testnet.hedera.com:50211": "0.0.6"
}});
```

On the Browser, it is:


```js
new Client({ network: {
  "grpc-web.myhbarwallet.com": "0.0.3",
}});
```

---

When making transactions or queries, a random node is picked from your local address book each time for client side load balancing.

Reply:
Great clarification, thanks! 😄

Reply:
bf418042bf409817a43a9011523c5511cbe0e072 should have updated _most_ of the examples to use this pattern @mehcode